### PR TITLE
Assignment 2.3

### PR DESCRIPTION
**Pull request by saachi-tiger**
Resolved issue #13 

- created new and activate the new branch
- added tests folder and created test_housing_price.py
- worked on setup the conda, build and install package and run the test using pytest

### CODE REVIEW

File: tests/test_housing_price.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+def test_load_housing_data(housing_data):
+    assert not housing_data.empty
+
+
+def test_income_cat_proportions(housing_data):
+    housing_data["income_cat"] = pd.cut(
+        housing_data["median_income"],
+        bins=[0.0, 1.5, 3.0, 4.5, 6.0, np.inf],
+        labels=[1, 2, 3, 4, 5],
+    )
```
Comment (abilashravi-ta) [inline tests/test_housing_price.py]: @saachi-tiger in this assignment (**2.3**) we have asked only for installation testing

**Review by abilashravi-ta — CHANGES_REQUESTED** [review]

**Review by abilashravi-ta — CHANGES_REQUESTED** [review]
Don't add the `__pycache__/` contents to the repo.

File: tests/test_housing_price.py (hunk 1 above)
Comment (saachi-tiger) [reply tests/test_housing_price.py]: Made changes to test_housing_price.py which test only the installation of package using pytest

File: .github/workflows/build_test.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Install the package
+      run: |
+        pip install dist/*.whl
+
+    - name: Run tests with pytest
+      run: |
+        pytest tests/
+      env:
+        INPUT_RUN_POST: true
+        CONDA_PKGS_DIR: /home/runner/conda_pkgs_dir
```
Comment (abilashravi-ta) [inline .github/workflows/build_test.yml]: why are these lines of code needed.?

**Review by abilashravi-ta — CHANGES_REQUESTED** [review]
Remove all the `__pycache__/` contents from the repo.

**Review by abilashravi-ta — APPROVED** [review]

File: .github/workflows/build_test.yml (hunk 2 above)
Comment (saachi-tiger) [reply .github/workflows/build_test.yml]: removed the unnecessary code

**Pull request merged**